docs/reference/index-modules.asciidoc

```diff
@@ -256,6 +256,10 @@ Other index settings are available in index modules:
     Settings to define analyzers, tokenizers, token filters and character
     filters.
 
+[xpack]#<<ilm-settings,{ilm-cap}>>#::
```

Comment:
The role attribute here generates

`<dt><span class="term"><span class="xpack xpack"><a class="link" href="ilm-settings.html" title="Index lifecycle management settings">Index lifecycle management</a></span></span></dt>`

... but we don't have anything for that in the https://github.com/elastic/docs/blob/master/resources/web/styles.css so  it doesn't have any effect.

Reply:
I moved it to a separate floated section so I could apply the attribute. I opened https://github.com/elastic/docs/issues/795 as a possible enhancement.